.github/workflows/ci.yml

```diff
@@ -88,15 +88,16 @@ jobs:
       - name: Run the tests
         run: docker run -v $(pwd):/tmp/rust-tss-esapi -w /tmp/rust-tss-esapi/tss-esapi --security-opt seccomp=unconfined fedoracontainer dbus-run-session -- /tmp/rust-tss-esapi/tss-esapi/tests/all-fedora.sh
 
-  tests-valgrind:
```

Comment:
I think we could add
```bash
if [[ ! -z ${RUST_TOOLCHAIN_VERSION:+x} ]]; then
	rustup override set ${RUST_TOOLCHAIN_VERSION}
fi
```

To the valgrind script file and then specify the  tool chain with a version we know works.